### DISCUSSION TITLE
How do I remove the Powered By... overlay in my short links?

### DISCUSSION THREAD

**Discussion by attentionsoft** [YOURLS/YOURLS#2988] [Q&A] [answered]
I just installed YOURLS and I noticed all of the shared links have an overlay that says "Short link powered by YOURLS and created X days ago, X views." How can this be removed?

![image](https://user-images.githubusercontent.com/18579114/125212998-9daf0780-e265-11eb-933e-fd27f22de792.png)

**Comment by dgw** [comment, marked as answer]
You have the `sample-toolbar` plugin enabled. Turn it off.

**Reply by attentionsoft** [reply]
That did the trick. Thank you!

**Reply by xuhongzhang123** [reply]
@dgw  how to Turn it off.

**Reply by dgw** [reply]
@xuhongzhang123 YOURLS has a "Manage plugins" page in the admin area, where you can activate/deactivate plugins. (The action link only appears when the mouse cursor is over that row.)

Sorry about the delay; this thread got buried in my mailbox!
